RecoTracker/LSTCore/interface/alpaka/Common.h

```diff
@@ -40,50 +41,49 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
 
   // The constants below are usually used in functions like alpaka::math::min(),
   // expecting a reference (T const&) in the arguments. Hence,
-  // ALPAKA_STATIC_ACC_MEM_GLOBAL needs to be used in addition to constexpr.
+  // HOST_DEVICE_CONSTANT needs to be used in addition to constexpr.
```

Comment:
This comment can probably be removed (or changed) as well.

Reply:
Yes, thanks for catching it.